CriteoPublisherSdk/Tests/CriteoPublisherSdk.xctestplan

```diff
@@ -10,7 +10,7 @@
   ],
   "defaultOptions" : {
     "codeCoverage" : false,
-    "language" : "fr-FR",
+    "language" : "fr-CA",
```

Comment:
Curious about this change. Reading the commit message:

> Xcode UI does not handle “fr-FR” but only “fr”

However, this change uses `fr-CA`.

Reply:
Xcode has a visual editor for there files that crashes with `fr-FR`, as the user interface has "French" (fr) and "French - Canada" (fr-CA), but hasn't `fr-FR` (French - France), figured this while trying to edit that file to add the launch argument to enable debug logs on tests.